content/en/platform/corda/4.8/open-source/key-concepts-node.md

```diff
@@ -20,31 +20,30 @@ title: Nodes
 
 ## Summary
 
-* *A node is JVM run-time with a unique network identity running the Corda software*
-* *The node has two interfaces with the outside world:*
-  * *A network layer, for interacting with other nodes*
-  * *RPC, for interacting with the node’s owner*
-* *The node’s functionality is extended by installing CorDapps in the plugin registry*
+* A node is JVM run-time with a unique network identity running the Corda software.
+* The node has two interfaces with the outside world:
+  * A network layer, for interacting with other nodes
+  * RPC, for interacting with the node’s owner
+* The node’s functionality is extended by installing CorDapps in the plugin registry.
 
 ## Video
 
 {{% vimeo 214168860 %}}
 
 ## Node architecture
 
-A Corda node is a JVM run-time environment with a unique identity on the network that hosts Corda services and
-CorDapps.
-
-We can visualize the node’s internal architecture as follows:
+A Corda node is a JVM runtime environment with a unique network identity. A node hosts Corda services and
```

Comment:
```suggestion
A Corda node is a JVM runtime environment with a unique network identity. A node hosts Corda services and
```
I'd add a brief explanation here of what a JVM runtime environment is. I'd also link to the ID header in the network doc.